arangod/Pregel/Actor/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_library(arango_actor INTERFACE)
```

Comment:
?
```suggestion
add_library(arango_pregel_actor INTERFACE)
```

Reply:
The actor is currently added to be used in the pregel system but it is not dependent on pregel in any way and could possibly be used in other arangodb code as well, therefore I like arango_actor better.